echo-api/src/main/java/com/netflix/spinnaker/echo/api/events/Event.java

```diff
@@ -25,11 +24,9 @@
 @Data
 public class Event {
   public Metadata details;
-  public Map content;
+  public Map<String, Object> content;
   public String rawContent;
-
-  @JsonInclude(JsonInclude.Include.NON_NULL)
-  public Map payload;
+  public Map<String, Object> payload;
 
   public String eventId = UUID.randomUUID().toString();
 }
```

Comment:
Is there value in `content` and `payload` being typed? I'd really like to avoid bringing `M<S, O>` into the `-api` modules if possible. Perhaps for Echo, we can't really do anything about it until the other services sending events in are well-defined?

Reply:
I think I'll merge as is and then explore how `content` and `payload` could be typed.  There's a lot to unwind WRT how `Event` and `TriggerEvent` (and the various `TriggerEvent` types) deserialize the `content` field and I don't feel comfortable making any changes there yet.